utils/camtrap_prep_1.py

```diff
@@ -214,6 +243,12 @@ def prep_camtrap_dp(file_path_raw:str=None, data_input:dict=None):  # sd.SdXDevi
         output_path = deploy_dir,
         )
 
+    # Generate gsheets observations.CSV
+    generate_gsheets_observations_datasets(
+        media_table = media_data,
```

Comment:
Tiny fix -- could you change `media_table` to `media_data` here in line 248.
Otherwise looks good to me!